sdk/data_cosmos/src/resource_quota.rs

```diff
@@ -81,11 +81,7 @@ pub(crate) fn resource_quotas_from_str(
         } else if let Some(stripped) = token.strip_prefix(AUTH_POLICY_ELEMENTS) {
             v.push(ResourceQuota::AuthPolicyElements(parseu64(stripped)?));
         } else {
-            return Err(Error::with_message(ErrorKind::DataConversion, || {
```

Comment:
I can't exactly tell from context, but is this quote string returned by the service or from input by the caller?

Reply:
Do you mean the quota string that's being parsed by this method? If so, it comes from Cosmos (apparently part of the response headers to certain requests based current usages of `resource_quotas_from_str`).

Reply:
Thanks. In case I agree it shouldn't be causing an error for callers.